.gitmodules

```diff
@@ -84,3 +84,7 @@
 [submodule "src/plugins/intel_cpu/thirdparty/libxsmm"]
 	path = src/plugins/intel_cpu/thirdparty/libxsmm
 	url = https://github.com/libxsmm/libxsmm.git
+[submodule "src/plugins/intel_cpu/thirdparty/shl"]
+	path = src/plugins/intel_cpu/thirdparty/shl
+	url = https://github.com/a-sidorova/csi-nn2.git
```

Comment:
Just a note based on offline discussion: it is not the best choice to use own dev fork as a thirdparty. 
Ideally we should try to upstream required changes. If this is not feasible lets create official fork inside openvinotoolkit group.

Reply:
Can we at least organize it the same way we do it for oneDNN?
Instead of "dev" branch, use 2.9.0_for_ie_master (I guess we better to refresh the naming we use)
And also have a clear explanation for the fork commits regarding why they are necessary
@dmitry-gorokhov @a-sidorova

Reply:
Agree.
We will create the new master branch `2.90_for_ie_master` in the new repository in openvinotoolkit group. At the moment, we're waiting for the repo creation: ticket 141759